src/ranged.cpp

```diff
@@ -630,205 +670,109 @@ void game::throw_item( player &p, const tripoint &target, item &thrown,
         deviation += rng(0, 3);
     }
 
-    deviation += rng(0, std::max( 0, p.str_cur - thrown.weight() / 113 ) );
+    deviation += rng(0, std::max( 0, str_cur - thrown.weight() / 113 ) );
 
-    double missed_by = .01 * deviation * trange;
-    bool missed = false;
-    int tart1, tart2;
-    bool do_railgun = (p.has_active_bionic("bio_railgun") &&
-            (thrown.made_of("iron") || thrown.made_of("steel")));
+    // Rescaling to use the same units as projectile_attack
+    const double shot_dispersion = deviation * (.01 / 0.00021666666666666666);
 
-    if (missed_by >= 1) {
-        // We missed D:
-        // Shoot a random nearby space?
-        if (missed_by > 9.0) {
-            missed_by = 9.0;
-        }
+    /*
+    // This causes crashes for some reason
+    static const std::vector<std::string> ferric = {{
+        "iron", "steel"
+    }};
+    */
+    std::vector<std::string> ferric;
```

Comment:
I've just seen, you use it as argument to `item::made_of_any`, which requires a `std::vector`, so it must be a vector. Strange, the static vector should work fine, it's already used that way in `iuse::solder_weld`.